test/test_cuda.py

```diff
@@ -4441,9 +4441,6 @@ def power2_div(size, div_factor):
         reg_mem = torch.cuda.memory_stats()[key_allocated]
         self.assertEqual(reg_mem - start_mem, nbytes)
 
-        with self.assertRaises(RuntimeError):
```

Comment:
This should still error out?

Reply:
Since foo:1,bar:2 may represent key-value configuration pairs for other accelerator backends, raising an error is inappropriate if `CUDAAllocatorConfig` does not recognize them.